### PR TITLE
feat:-add modal component

### PR DESCRIPTION
**Pull request by shubhamkumar648**
modal component
1.basic live
modal2
modal
 modal

### CODE REVIEW

**Review by aniketsingh13 — APPROVED** [review]
Good to go

File: javascript/modal.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+console.log(modelAvtive);
+
+modelAvtive.addEventListener("click", () => {
+    modelBox.style.display="block"
+})
+
+modalClose.addEventListener('click', () => {
+    modelBox.style.display="none"
+
+})
```
Comment (rahulcs754) [inline javascript/modal.js]: here you use es5 and es6 that is i don't like you should use es6 all or es5 only one. that is not good.
please remove extra space

**Review by rahulcs754 — COMMENTED** [review]
I looked at your HTML code other code.
I have given some suggestions you should check.

File: javascript/modal.js (hunk 1 above)
Comment (shubhamkumar648) [reply javascript/modal.js]: ok thank you for suggestion ! i will do

**Review by rahulcs754 — APPROVED** [review]

**Comment by shubhamkumar648** [comment]
> I looked at your HTML code other code. I have given some suggestions you should check.

done

**Pull request merged**